tests/utils/verification/volumeproperties.go

```diff
@@ -161,3 +163,52 @@ func GetAssociatedPolicyName(hostname string, volName string) (string, error) {
 	}
 	return op, err
 }
+
+// CheckVolumeInProcMounts - check if volume entry is present in docker host mount file
+// grep does the work. If entry isn't present, err is not nil (Exit status 1)
+func CheckVolumeInProcMounts(hostName, volName string) bool {
```

Comment:
Need to find a better home for newly added methods .. it spoils the objective of this current file.